### DISCUSSION TITLE
Deploy lock found

### DISCUSSION THREAD

**Discussion by mateuszbialowas** [basecamp/kamal#262] [Q&A]
Hi, 
I try to deploy my new app with mrsk. 
This is my Docerfile, docker-entrypoint, deploy.yml files 
https://gist.github.com/mateuszbialowas/059c916cde434b809ff7514405194fc8

When I run:
mrsk deploy then I have error
  ERROR (Mrsk::Cli::LockError): Deploy lock found

How to fix this?
<img width="1030" alt="CleanShot 2023-05-02 at 22 49 07@2x" src="https://user-images.githubusercontent.com/58574619/235782604-5f4f9f44-186b-4078-a0a5-073ca55974ad.png">

**Comment by intrip** [comment]
You can run `mrsk lock release` to force realising the lock.

**Reply by mateuszbialowas** [reply]
Thanks:
This is correct command
`mrsk lock release`

**Reply by dorianmariecom** [reply]
So `kamal lock release` now

**Comment by guimon** [comment]
As of 10/23/23, the updated command is `kamal lock release`

**Reply by clint326** [reply]
Thanks so much for posting the above answer! I was getting the 'ERROR (Kamal::Cli::LockError): Deploy lock found' message and couldn't figure it out.
